packages/os/os.spec

```diff
@@ -315,15 +315,41 @@ migrations=()
 for migration in $(find %{_builddir}/sources/api/migration/migrations/v[0-9]* -mindepth 1 -maxdepth 1 -type d); do
     migrations+=("-p $(basename ${migration})")
 done
+
 # Store the output so we can print it after waiting for the backgrounded job.
+exec 3>&1 4>&2
 static_output="$(mktemp)"
+exec 1>"${static_output}" 2>&1
 # Build static binaries in the background.
 %cargo_build_static --manifest-path %{_builddir}/sources/Cargo.toml \
     -p apiclient \
     ${migrations[*]} \
-    >> ${static_output} 2>&1 &
+    &
 # Save the PID so we can wait for it later.
 static_pid="$!"
+exec 1>&3 2>&4
+
+%if %{with aws_platform} || %{with aws_k8s_family}
+# The AWS SDK crates are extremely slow to build with only one codegen unit.
+# Pessimize the release build for just the crates that depend on them.
+# Store the output so we can print it after waiting for the backgrounded job.
+exec 3>&1 4>&2
+aws_sdk_output="$(mktemp)"
+exec 1>"${aws_sdk_output}" 2>&1
+RUSTFLAGS="$(echo "%{__global_rustflags_shared}" | sed -e 's,-Ccodegen-units=1,,g')" \
```

Comment:
Just for my own understanding, what's the penalty of building all the first-party crates with more than one code unit?

Reply:
> Just for my own understanding, what's the penalty of building all the first-party crates with more than one code unit?

I expect it'd be in the realm of 10-20% binary size increase and an unknown performance hit. It'd push us pretty close to the current filesystem size limit for existing variants.